pkg/orders/orders.go

```diff
@@ -0,0 +1,30 @@
+package orders
+
+import (
+	"context"
+
+	"github.com/clearchanneloutdoor/io-sdk-golang/pkg/clients"
+	"golang.org/x/oauth2/clientcredentials"
+)
+
+const (
+	serverUrl string = "https://direct.cco.io"
+)
+
+func NewOrdersClient(ctx context.Context, oauth2 *clientcredentials.Config, overrideSvr ...string) (*clients.Client[Order], error) {
+	svr := serverUrl
+	if len(overrideSvr) > 0 && overrideSvr[0] != "" {
+		svr = overrideSvr[0]
+	}
+
+	return clients.NewClient[Order](ctx, svr, "/v1/orders", oauth2)
+}
+
+func NewOrderLinesClient(ctx context.Context, oauth2 *clientcredentials.Config, overrideSvr ...string) (*clients.Client[OrderLine], error) {
+	svr := serverUrl
+	if len(overrideSvr) > 0 && overrideSvr[0] != "" {
+		svr = overrideSvr[0]
+	}
+
+	return clients.NewClient[OrderLine](ctx, svr, "/v1/orderlines", oauth2)
```

Comment:
do any of these need any scopes added? not sure what the use case is so they might not

Reply:
We're not currently enforcing any scope on order-api or booking-api... some oversight on our end but it's coming down the line. i'll update these when we start to enforce them.